input/system/rds/logs.go

```diff
@@ -172,8 +120,126 @@ func DownloadLogFiles(server *state.Server, logger *util.Logger) (state.Persiste
 
 ErrorCleanup:
 	for _, logFile := range logFiles {
-		logFile.Cleanup()
+		logFile.Cleanup(logger)
 	}
 
 	return server.LogPrevState, nil, nil, err
 }
+
+var DescribeDBClustersErrorCache *util.TTLMap = util.NewTTLMap(10 * 60)
+
+func getAwsDbInstanceID(config config.ServerConfig, sess *session.Session) (string, error) {
+	identifier := config.AwsDbInstanceID
+	// If this is an Aurora cluster endpoint, we need to find the actual instance ID in order to get the logs
+	if identifier == "" {
```

Comment:
NBD, but I think an early return here is preferable to putting most of the method behind an if condition.